contacts.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="ru-RU">
```

Comment:
Why do you write lang="ru", while the content you mostly in English?

Reply:
I probably didn't think much of it.